src/index.ts

```diff
@@ -0,0 +1,6 @@
+import * as ts from 'typescript/lib/typescript'
```

Comment:
This index file is not needed

Reply:
Originally main was a "lib/typescript.js" in package.json. Now "main": "lib/index.js". Better to change it back in package.json or keep standard index.js?